style-guide.rst

```diff
@@ -0,0 +1,717 @@
+ - easily
+ - just
+ - very
+ - really
+ - basically
+
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  To open the file, simply click the button.
+  
+.. rubric:: Right
+
+.. code-block:: rst
+
+  To open the file, click the button.
+  
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  You can easily edit the form by...
+  
+.. rubric:: Right
+
+.. code-block:: rst
+
+  To edit the form...
+  
+.. _filler-phrases:  
+  
+Filler words and phrases
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Many words and phrases provide no direct meaning. They are often inserted to make a sentence seem more formal, or to simulate a perceived style of business communication. These should be removed.
+
+Common filler phrases and words include:
+
+- to the extent that
+- for all intents and purposes
+- when all is said and done
+- from the perspective of
+- point in time
+
+This list is not exhaustive. These "canned phrases" are pervasive in technical writing. Remove them whenever they occur.
+
+.. _semicolons:
+
+Semicolons
+-------------
+
+Semicolons are used to separate two independent clauses which could stand as individual sentences but which the writer feels would benefit by close proximity.
+
+Semicolons can almost always be replaced with periods (full stops). This rarely diminishes correctness and often improves readability.
+
+.. rubric:: Correct
+
+.. code-block:: rst
+
+  These "canned phrases" are pervasive in technical writing; remove them whenever they occur.
+  
+.. rubric:: Better
+
+.. code-block:: rst
+
+  These "canned phrases" are pervasive in technical writing. Remove them whenever they occur.
+
+.. _pronouns:
+    
+Pronouns
+----------
+
+.. _third-person-pronouns:
+
+Third-person personal pronouns
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Third-person personal pronouns are:
+
+- he/him/his
+- she/her/her(s)
+- they/them/their(s) 
+
+.. note:: 
+
+  While some people consider *they/them/their* to be non-standard (or "incorrect") as third-person singular, it has gained wide use as a gender-neutral or gender-ambigious alternative to *he* or *she*.
+
+There are two issues with personal pronouns:
+
+- gender bias
+- clarity
+
+To avoid gender bias, the third person gender-neutral *they/then/their(s)* is preferred over *he* or *she* pronouns when writing about abstract individuals.
+
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  The enumerator uses his device.
+  
+.. rubric:: Right
+
+.. code-block:: rst
+
+  The enumerator uses their device.
+
+
+Unfortunately, *they/them/their* is not a perfect solution. Since it is conventionally used as a plural pronoun, it can cause confusion.
+
+Therefore, avoid the use of personal pronouns whenever possible. They are often out of place in technical writing anyway. Rewriting passages to avoid personal pronouns often makes the writing more clear.
+
+.. rubric:: Correct
+
+.. code-block:: rst
+
+  When using Collect, first the enumerator opens the app on their device. Then they complete the survey.
+  
+.. rubric:: Improved
```

Comment:
You used Improved here, but Better everywhere else. Was that your intention?

Reply:
No. I'm just losing my mind is all.